identity/webapp/src/utility/configure-auth0.ts

```diff
@@ -22,7 +22,7 @@ export function configureAuth0(): void {
   koaPassport.deserializeUser(async function (user: any, done) {
     const accessToken: string = user?.accessToken;
     try {
-      await callAuth0Api('GET', '/userinfo', { user: { accessToken } });
+      await callRemoteApi('GET', '/users/me', { user: { accessToken } });
```

Comment:
Do we want to store the data we get back so we have the e-mail, etc. available later in the `ApplicationState`?

Reply:
We probably do yeah, but I'll add that in a future PR when we have a use for it.